test/e2e/reconciler/workspace/controller_test.go

```diff
@@ -331,9 +340,11 @@ func unschedulable(object *tenancyv1alpha1.ClusterWorkspace) error {
 func scheduled(target string) func(workspace *tenancyv1alpha1.ClusterWorkspace) error {
 	return func(object *tenancyv1alpha1.ClusterWorkspace) error {
 		if isUnschedulable(object) {
+			klog.Infof("workspace %s is unschedulable, skipping scheduled check", object.Name)
```

Comment:
Why do we need the klogs, should the errors not be logged by `t.Error`?

Reply:
Just to have some indication when things go wrong. Silent wait loops are usually silent despite the logs when things are good, but should be noisy when things are bad.